cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -370,14 +376,8 @@ open class Node @Inject constructor(private val project: Project) {
     }
 
     internal fun buildDocker() {
-        project.copy {
-            it.apply {
-                from(Cordformation.getPluginFile(project, "Dockerfile"))
```

Comment:
Should `/cordformation/src/main/resources/net/corda/plugins/Dockerfile` be removed as well?

Reply:
yup, removed!